src/react/features/rooms/effects/useGetHomeEffect.js

```diff
@@ -7,12 +7,22 @@ import { setConnections } from '../../connectors/actions'
 
 import { invokeChannel } from '../../../client/client'
 import { channels } from '../../../../shared/constants'
-
+import { invokeAlertDialog } from '../../../client/alertDialog'
+/*
+ * Requests home & focus info via GET_HOME and sets the focus for room & diory
```

Comment:
I'm not in favour for comments. The code itself or specs (tests) at least should say all this.

Reply:
If they does not, they should be improved. The need for comment typically means that the code is not good enough.

Reply:
Comments will typically be obsolete.